Dockerfile

```diff
@@ -4,6 +4,7 @@ COPY default.conf /etc/nginx/conf.d/
 COPY nginx.conf /etc/nginx/
 COPY docker-entrypoint.sh /
 RUN chmod +x /docker-entrypoint.sh
+ENV CLIENT_MAX_BODY_SIZE 20M
```

Comment:
Let's make this even bigger. Perhaps 256MB.

Reply:
The page where i found this configuration value stated that 20M is the default value.
I thought of adding a configuration value to the one-click app for this.

Reply:
Damn! I am wrong, its only 1M...
I took your suggestion of 256M and created another pull request for the oneclick-apps.

#https://github.com/caprover/one-click-apps/pull/59